homeassistant/components/config/config_entries.py

```diff
@@ -47,13 +49,22 @@ class ConfigManagerEntryIndexView(HomeAssistantView):
     async def get(self, request):
         """List flows in progress."""
         hass = request.app['hass']
+
+        def supports_options(domain):
+            """"""
+            if domain == 'axis':
+                return None
+            handler = config_entries.HANDLERS[domain]
```

Comment:
What about `hasattr(config_entries.HANDLERS[domain], 'async_get_options_flow')` ? Because initializing an option flow might require more work, which we don't want to do for just testing if it exists.

Reply:
Absolutely :+1

Reply:
I think it should initialize it, so that we can support a generic options handler that can be reused by various handlers.

Reply:
Allright. Removing async_get_options_flow from Config_Flow base class as well then